static/js/common.ts

```diff
@@ -52,7 +52,7 @@ const keys_map = new Map([
     ["PgUp", "↑"],
     ["PgDn", "↓"],
     ["Ctrl", "⌘"],
-    ["Alt", "⌘"],
+    ["Alt", "⌥"],
```

Comment:
Why does this later commit change this character -- probably that change should be its own commit with a commit message explaining the reasoning?

Reply:
I have shifted it to a different commit and added a commit message. The equivalent of "Alt" key in mac is the option key (⌥) and not the command key (⌘).

Reply:
So, even though this might be true in general, I think we still want to use `⌘` here.

Currently the only shortcut being documented with the `Alt` key is for navigating back and forward in the viewing/browser history. And for the major web browsers those are all documented as the command key with the arrow keys, see [Firefox documentation](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_navigation), [Chrome documentation](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts) and [Safari documentation*](https://support.apple.com/guide/safari/keyboard-and-other-shortcuts-cpsh003/mac)

I've tested on my (old) Mac laptop and `⌘` with the left/right arrow works for Zulip desktop and all the browsers noted above. While `⌥` with the left/right arrow does not navigate back and forth in any of the above or Zulip desktop.

I guess this could be an issue if/when we need to document keyboard shortcuts that use the `Alt` key and are mapped to the `⌥` key on Macs, but I don't think that's currently the case.

*Note the Safari documentation actually documents the shortcut as `Command` + `[` or `Command` + `]` which is a variant that doesn't use the arrow keys, but note that the command key is being used.*

Reply:
@laurynmm This makes sense to me, thanks. Since, as you mention, we don't have any other shortcuts that could rise to Alt vs Command debate, I'll take this commit off of this PR.